### DISCUSSION TITLE
Transfer Option (SFTP)

### DISCUSSION THREAD

**Discussion by EICP02** [EricZimmerman/KapeFiles#804] [Q&A]
Hi guys, 

Every time that I use the transfer option (SFTP), when it runs it create a directory (KAPE_data_push_PCNAME_DATE) in the destination even thought I use the parameters --scd (Blank), Is there a way to not to create that directory? 

If not, my suggestion is to create a way to do it. 

Context:
I'm trying to automate my acquisitions with KAPE and I want to sent the compress file (.ZIP) to my SFTP server. In my SFTP server I have a rule that sent every file to my forensics server but this rule only works with files, no directory.

**Comment by AndrewRathbun** [comment]
Maybe this is best brought up as an issue and not a discussion?

**Reply by EICP02** [reply]
Nice thanks, let me try.
